tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -744,15 +751,19 @@ Function RestorePreviousInstallLocation
     StrCpy $INSTDIR $4
 FunctionEnd
 
+Function Skip
+  Abort
+FunctionEnd
+
 Function SkipIfPassive
   ${IfThen} $PassiveMode = 1  ${|} Abort ${|}
 FunctionEnd
 
 Function CreateOrUpdateStartMenuShortcut
   ; We used to use product name as MAINBINARYNAME
   ; migrate old shortcuts to target the new MAINBINARYNAME
-  ${If} ${FileExists} "$DESKTOP\${PRODUCTNAME}.lnk"
-    !insertmacro SetShortcutTarget "$DESKTOP\${PRODUCTNAME}.lnk" "$INSTDIR\${MAINBINARYNAME}.exe"
+  ${If} ${FileExists} "$SMPROGRAMS\$AppStartMenuFolder\${PRODUCTNAME}.lnk"
```

Comment:
Shouldn't we also check for `$SMPROGRAMS\${PRODUCTNAME}.lnk`, and update it?

Reply:
Right, was thinking that we only have shortcuts with folders, forgot some people like myself had it without the folder 😂